packages/asgardex-bitcoin/src/client.ts

```diff
@@ -183,11 +197,12 @@ class Client implements BitcoinClient, AsgardexClient {
   // Given a desired output, return change
   private getChange = async (valueOut: number): Promise<number> => {
     const balances = await this.getBalance()
-    const btcBalance = balances.find((balance) => balance.coin === 'BTC.BTC')
+    //TODO(kashif) needs verification if this is how we should compare symbol === 'BTC.BTC'
+    const btcBalance = balances.find((balance) => balance.asset.symbol === 'BTC.BTC')
```

Comment:
Safest way to compare assets here ^:
```
assetToString(balance.asset) === assetToString(AssetBTC)
```

[`assetToString` is a helper](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/asset.ts#L170) provided by `asgardex-util`.

We already have [a `eqAsset` helper](https://github.com/thorchain/asgardex-electron/blob/58ec9b6885cf015759e6ff2daadf1f768c0474ab/src/renderer/helpers/fp/eq.ts#L16) in `asgardex-electron`, but that's not provided in `agardex-util` atm.